indexer-app/src/cmr/indexer/services/index_set_service.clj

```diff
@@ -234,10 +211,38 @@
     (if (contains? rebalancing-colls-set concept-id)
       (seq (disj rebalancing-colls-set concept-id))
       (errors/throw-service-error
-        :bad-request
-        (format
-          "The index set does not contain the rebalancing collection [%s]"
-          concept-id)))))
+       :bad-request
+       (format
+        "The index set does not contain the rebalancing collection [%s]"
+        concept-id)))))
+
+(defn- is-rebalancing?
```

Comment:
I'm not sure how these two functions are being used, but we allow multiple collections to be in the rebalancing list simultaneously. If these checks are being used to try and prevent multiple collections being updated there is an issue.